### ISSUE TITLE
Thaumarhia/SunScorned crashing clients

### ISSUE THREAD

**Issue by Gregolo** [Azanor/thaumcraft-5#340]
Wondered if you could shed some insight. Can't find documentation on others with the same problem.

Modpack: Infinity Evolved (Expert Mode)
Thaumcraft Version: 4.2.3.5

Whenever any of my users get potionID:84 they CTD themselves and everyone around them. This is confirmed via /effect name 84. Which can reproduce the issue. Only way to fix their profile is by removing potionID:84 in their active buffs within their NBT data. 

Doing an NEI PotionID dump I see that PotionID:83 is 'Thaumarhia' but the console will announce that it's 'Sun Scorned' when the /effect <name> 83 command is given. The dump also states that PotionID:84 is 'null' and therefore unused. Yet typing /effect <name> 84 crashes the client, but reports giving them 'Thaumarhia' for 30 seconds.

TL;DR:

/effect 83 is labelled as 'Thaumarhia' but gives 'Sun Scorned'
/effect 84 is labelled as 'null' but gives 'Thaumarhia' and makes people CTD

Additional Context:
[Crash Report] - http://pastebin.com/dsfhvtgW (ignore the name, she's cheeky)
[Additional Mods] - Cosmetic stuff like BigTrees, Ztones, Malisisdoors. Shouldn't affect potionIDs?

**Comment by Azanor** [comment]
You need to take this up with the modpack creator since it is obviously a config issue. Secondly Thaumcraft 4 is no longer supported.
